NetflixClone/Classes/Scenes/Home/HomeViewController.swift

```diff
@@ -192,8 +197,13 @@ extension HomeViewController: UICollectionViewDelegate {
         let items = snapshot.itemIdentifiers(inSection: section)
         let movie = items[indexPath.row]
 
-        let viewController = MovieDetailsViewController(movie: movie)
-        present(viewController, animated: true)
-
+//        let viewController = MovieDetailsViewController(movie: movie)
+//        present(viewController, animated: true)
+        
+        let infoViewModel = MovieInfoViewModel(id: movie.id,title: movie.title, posterPath: movie.posterPath, description: movie.overview, rating: movie.voteAverage, date: movie.releaseDate)
```

Comment:
It's not recommended to add multiple parameters to a method or an initializer... A good solution might be creating a new struct to hold all needed inputs 🤝🏼